docs/pages/admin-guides/management/diagnostics/metrics.mdx

```diff
@@ -14,12 +23,168 @@ collect from your Teleport cluster.
+Teleport-protected resources flows through the Proxy Service, an Agent, and
+finally the infrastructure resource the Agent proxies. Return traffic from the
+resource takes this path in reverse.
+
+### Number of connected resources by type
+
+Teleport-connected resources periodically send heartbeat (keepalive) messages to
+the Auth Service. The Auth Service uses these heartbeats to track the number of
+Teleport-protected resources by type with the `teleport_connected_resources`
+metric. 
+
+The Auth Service tracks this metric for the following resources:
+
+- SSH servers
+- Kubernetes clusters
+- Applications
+- Databases
+- Teleport Database Service instances
+- Windows desktops
+
+You can use this metric to:
+- Compare the number of resources that are protected by Teleport with those that
+  are not so you can plan your Teleport rollout, e.g., by configuring [Auto
+  Discovery](../../../enroll-resources/auto-discovery/auto-discovery.mdx).
+- Correlate changes in Teleport usage with resource utilization on Auth Service
+  and Proxy Service compute instances to determine scaling needs.
+
+You can include this query in your Grafana configuration to break this metric
+down by resource type:
+
+```text
+sum(teleport_connected_resources) by (type)
+```
+
+### Reverse tunnels by type
+
+Every Teleport service that starts up establishes an SSH reverse tunnel to the
+Proxy Service. (Self-hosted clusters can configure Agent services to connect to
+the Auth Service directly without establishing a reverse tunnel.) The Proxy
+Service tracks the number of reverse tunnels using the metric,
+`teleport_reverse_tunnels_connected`.
+
+With an improperly scaled Proxy Service pool, the Proxy Service can become a
+bottleneck for traffic to Teleport-protected resources. If Proxy Service
+instances display heavy utilization of compute resources while the number of
+connected infrastructure resources is high, you can consider scaling out your
+Proxy Service pool and using [Proxy Peering](../operations/proxy-peering.mdx).
+
+Use the following Grafana query to track the maximum number of reverse tunnels
+by type over a given interval:
+
+```text
+max(teleport_reverse_tunnels_connected) by (type))
+```
+
+### Count and version of Teleport Agents
+
+Alongside the number of connected resources and reverse tunnels, you can track
+the number of Agents in your Teleport cluster. Since you can run multiple
+Teleport services on a single Agent instance, this metric helps you understand
+the architecture of your Teleport Agent deployment so you can diagnose issues
+with resource utilization.
+
+At regular intervals (around 7 seconds with jitter), the Auth Service refreshes
+its count of registered Agents. You can measure this count with the metric,
+`teleport_registered_servers`. To get the number of registered Agents by
+version, you can use this query in Grafana:
 
-<Notice scope={["cloud"]} type="tip">
+```text
+sum by (version)(teleport_registered_servers)
```

Comment:
Just a note, this metric will also count any registered proxies and auths connected.  If someone wanted to count only agents they'd want to subtract their auth/proxies.

Reply:
👍 I've updated this section to focus on versioning, since (a) we already cover Agent resource utilization and (b) there doesn't seem to be a way to group this metric by service